internal/utils/stat.go

```diff
@@ -74,6 +75,10 @@ func Stat(ctx context.Context, store *multistore.Store, root cid.Cid, sel ipld.N
 	}.WalkMatching(nd, s, func(prog traversal.Progress, n ipld.Node) error {
 		return nil
 	})
+	if err != nil {
+		return res, err
```

Comment:
```suggestion
		return nil, err
```

Reply:
does not work because res is not a pointer